src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Ball {
+    private String color;
+    private int number;
+
+    public String getColor() {
+        return color;
+    }
+
+    public void SetColor(String color) {
+        this.color = color;
+    }
```

Comment:
The method name 'SetColor' does not follow Java naming conventions. Method names should begin with a lowercase letter. Please rename this to 'setColor'.